website/content/SDK/economy/_index.md

```diff
@@ -1,6 +1,111 @@
 ---
-title: economy
-keywords: ""
-excerpt: ""
+title: Introduction to the OpenBB SDK Economy Module
```

Comment:
Rename to "Introduction to Economy", we don't have to highlight this is for the SDK.